### PR TITLE
Fix org query set

### PR DESCRIPTION
**Pull request by koutst**
- The Organizations queryset wasn't filtering Organizations by user so a user could get access to all organizations.

### CODE REVIEW

File: src/user/views/organization_view.py [hunk 1]
```diff
@@ -124,7 +131,10 @@ def _create_image_file(self, data, organization, user):
     def update(self, request, *args, **kwargs):
         user = request.user
         partial = kwargs.pop("partial", False)
-        organization = self.get_object()
+        try:
+            organization = self.get_object()
+        except Exception:
+            return Response({"data": "No permission to get organization"}, status=403)
```
Comment (gzurowski) [inline src/user/views/organization_view.py:137]: I am wondering if we can assume an access issue for any type of exception?

**Review by gzurowski — APPROVED** [review]
LGTM 👍

File: src/user/views/organization_view.py (hunk 1 above)
Comment (yattias) [reply src/user/views/organization_view.py:137]: I think so

**Review by yattias — APPROVED** [review]
This looks good

File: src/user/views/organization_view.py (hunk 1 above)
Comment (koutst) [reply src/user/views/organization_view.py:137]: `get_object` returns an Http404 error if not found so I will just check for that.

Comment (gzurowski) [reply src/user/views/organization_view.py:138]: The approach can mask legitimate _not found_ cases, i.e., where a non-existing org is requested. I believe a HTTP 404 response for both cases should be fine.

**Pull request merged**